core/autoload/spacevim/plug/fzf.vim

```diff
@@ -190,6 +190,34 @@ function! s:align_lists(lists)
   return a:lists
 endfunction
 
+autocmd VimEnter * command! -bang -nargs=* Ag
```

Comment:
`autocmd VimEnter *` is useless in the `autoload` mechanism. So, remove `autocmd VimEnter *`.